Master table database scripts/College Tiering Script.txt

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE CollegeTiering 
```

Comment:
Use Master tables with format of "MASTER" as prefix. EX: MASTER_COLLEGE_TIERING , MASTER_GENER,MASTER_OFFERED_BAND.
Change this in all places.